autoload/lightline/colorscheme/srcery.vim

```diff
@@ -6,35 +6,29 @@
 " Last Modified: 2017-03-28
 " -----------------------------------------------------------------------------
 
-function! s:getSrcColor(group) abort
-  let l:guiColor = synIDattr(hlID(a:group), 'fg', 'gui')
-  let l:termColor = synIDattr(hlID(a:group), 'fg', 'cterm')
-  return [ l:guiColor, l:termColor ]
-endfunction
-
 if exists('g:lightline')
 
-  let s:black        = s:getSrcColor('SrceryBlack')
-  let s:bright_white = s:getSrcColor('SrceryBrightWhite')
-  let s:bright_black = s:getSrcColor('SrceryBrightBlack')
-  let s:white        = s:getSrcColor('SrceryWhite')
-  let s:yellow       = s:getSrcColor('SrceryYellow')
-  let s:blue         = s:getSrcColor('SrceryBlue')
-  let s:bright_blue  = s:getSrcColor('SrceryBrightBlue')
-  let s:cyan         = s:getSrcColor('SrceryCyan')
-  let s:bright_cyan  = s:getSrcColor('SrceryBrightCyan')
-  let s:red          = s:getSrcColor('SrceryRed')
-  let s:bright_red   = s:getSrcColor('SrceryBrightRed')
-  let s:orange       = s:getSrcColor('SrceryOrange')
-  let s:magenta      = s:getSrcColor('SrceryMagenta')
-  let s:green        = s:getSrcColor('SrceryGreen')
-  let s:bright_green = s:getSrcColor('SrceryBrightGreen')
-  let s:xgray1       = s:getSrcColor('SrceryXgray1')
-  let s:xgray2       = s:getSrcColor('SrceryXgray2')
-  let s:xgray3       = s:getSrcColor('SrceryXgray3')
-  let s:xgray4       = s:getSrcColor('SrceryXgray4')
-  let s:xgray5       = s:getSrcColor('SrceryXgray5')
-  let s:xgray6       = s:getSrcColor('SrceryXgray6')
+  let s:black        = srcery#helper#GetColor('SrceryBlack')
```

Comment:
Nitpick, maybe sort them as the others?  Believe they follow the xterm colors code.

Reply:
sorted